02/Form.js

```diff
@@ -2,9 +2,10 @@ import React from 'react';
 
 class Form extends React.Component {
     render() {
+        const {onClick} = this.props;
         return (
             <section>
-                <input /><input type="submit" />
+                <input ref={node => this.input = node}/><input type="submit" onClick={onClick}/>
```

Comment:
Pamiętaj, że może też zrobić coś na zasadzie `onClick = {() => { this.onClick(this.input.value) }}` - tutaj też możesz resetować ten input